tests/unit/lms/validation/authentication/_bearer_token_test.py

```diff
@@ -103,23 +107,14 @@ def schema(self, pyramid_request):
         return BearerTokenSchema(pyramid_request)
 
     @pytest.fixture
-    def pyramid_request(self, _jwt):
-        pyramid_request = DummyRequest()
-        pyramid_request.headers[
-            "authorization"
-        ] = f"Bearer {_jwt.encode_jwt.return_value}"
+    def pyramid_request(self, pyramid_request):
+        pyramid_request.headers["authorization"] = "Bearer ENCODED_JWT"
         return pyramid_request
 
 
 @pytest.fixture(autouse=True)
-def _jwt(patch, lti_user):
-    _jwt = patch("lms.validation.authentication._bearer_token._jwt")
-    _jwt.encode_jwt.return_value = "ENCODED_JWT"
-    _jwt.decode_jwt.return_value = lti_user._asdict()
-    return _jwt
-
+def jwt_service(jwt_service, lti_user):
+    jwt_service.decode_with_secret.return_value = lti_user._asdict()
+    jwt_service.encode_with_secret.return_value = "ENCODED_JWT"
 
-@pytest.fixture
-def lti_user():
-    """Return the original LTIUser that was encoded as a JWT in the request."""
-    return factories.LTIUser()
+    return jwt_service
```

Comment:
Some of the tests in this file don't seem like they are testing this class / are duplicates.

* `test_serialize_and_deserialize_via_marshmallow_api` - Seems like a test of our base class
* `test_parse_via_webargs_api` - Seems like a test of marshmallow / a duplicate of the first test

There's also a chunk of code which very nearly the same in 4 tests which is:

```python
        with pytest.raises(error) as exc_info:
            schema.lti_user("headers")

        assert exc_info.value.messages == {
            "headers": {field: [message]},
        }
```

A small helper there + some parameterisation might go a long way to shortening the test.